portfolio_home.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+
+<html lang="en-GB">
+
+	<head>
+		<title>Welcome to Karen Schulz's Page</title>
+		<link rel="stylesheet" type="text/css" href="portfolio_style.css">
+	</head>
+		
+	<body>
+		
+		<nav class="nav_bar">
+			<div>
+				<label class="label_karen_schulz" for="nav_bar">Karen Schulz</label>
+			</div>
+			<div>
+				<a href="portfolio.html">Portfolio</a>
+				<a href="contact.html">Contact</a>
+			</div>
+		</nav>
+		
+		<div class="welcome">
+			
+			<img class="mypicture" src="images/profile_pic.png" alt="my_profile_picture">
```

Comment:
in alt describe your profile pic, hair style, accessories, mood etc

Reply:
oh... really? it's quite funny! But I did what you suggested